File: D4office/d4office-core/src/main/java/com/dedalus/d4office/entity/Office.java
package com.dedalus.d4office.entity;


import java.sql.Blob;
import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Table(name = "offices")
@Entity
public class Office {	
	
	@Id
	@Column(nullable = false)
	private String officeId;	
	private String officeDesc;	
	private Double xDim;
	private Double yDim;
	private Double yOnXRatio;
	private Double yLenght;
	private Double radiusScaleFactor;
	private Blob officeLogo;
	private Blob officePlan;
	private String address;
}

File: D4office/d4office-web/src/app/office.ts
export class Office {
    officeId: String = "";	
	officeDesc: String = "";	
	xDim: Number = 0;
	yDim: Number = 0;
	yOnXRatio: Number = 0;
	yLenght: Number = 2000;
	radiusScaleFactor: Number = 0;

    public getYLenght(): Number {
        return this.yLenght;
    }
}

